dm/metrics/alertmanager/dm_worker.rules.yml

```diff
@@ -68,33 +68,33 @@ groups:
       summary: DM fail to write relay log
 
   - alert: DM_dump_process_exists_with_error
-    expr: changes(dm_mydumper_exit_with_error_count[1m]) > 0
+    expr: changes(dm_mydumper_exit_with_error_count{resumable_err="false"}[1m]) > 0 or on(source_id, task) increase(dm_mydumper_exit_with_error_count{resumable_err="true"}[2m]) > 3
     labels:
       env: ENV_LABELS_ENV
       level: critical
-      expr: changes(dm_mydumper_exit_with_error_count[1m]) > 0
+      expr: changes(dm_mydumper_exit_with_error_count{resumable_err="false"}[1m]) > 0 or on(source_id, task) increase(dm_mydumper_exit_with_error_count{resumable_err="true"}[2m]) > 3
     annotations:
       description: 'cluster: ENV_LABELS_ENV, instance: {{ $labels.instance }}, task: {{ $labels.task }}, values: {{ $value }}'
       value: '{{ $value }}'
       summary: DM dump process exists with error
 
   - alert: DM_load_process_exists_with_error
-    expr: changes(dm_loader_exit_with_error_count[1m]) > 0
+    expr: changes(dm_loader_exit_with_error_count{resumable_err="false"}[1m]) > 0 or on(source_id, task) increase(dm_loader_exit_with_error_count{resumable_err="true"}[2m]) > 3
     labels:
       env: ENV_LABELS_ENV
       level: critical
-      expr: changes(dm_loader_exit_with_error_count[1m]) > 0
+      expr: changes(dm_loader_exit_with_error_count{resumable_err="false"}[1m]) > 0 or on(source_id, task) increase(dm_loader_exit_with_error_count{resumable_err="true"}[2m]) > 3
     annotations:
       description: 'cluster: ENV_LABELS_ENV, instance: {{ $labels.instance }}, task: {{ $labels.task }}, values: {{ $value }}'
       value: '{{ $value }}'
       summary: DM load process exists with error
 
   - alert: DM_sync_process_exists_with_error
-    expr: changes(dm_syncer_exit_with_error_count[1m]) > 0
+    expr: changes(dm_syncer_exit_with_error_count{resumable_err="false"}[1m]) > 0 or on(source_id, task) increase(dm_syncer_exit_with_error_count{resumable_err="true"}[2m]) > 3
     labels:
       env: ENV_LABELS_ENV
       level: critical
-      expr: changes(dm_syncer_exit_with_error_count[1m]) > 0
+      expr: changes(dm_syncer_exit_with_error_count{resumable_err="false"}[1m]) > 0 or on(source_id, task) increase(dm_syncer_exit_with_error_count{resumable_err="true"}[2m]) > 3
```

Comment:
change `expr` to `increase(xxxxxxx[2m]) > 3` in case the resumeable error is a `connection timeout` which takes 30s, @lance6716 ptal

Reply:
I'm not sure the pause and resume action run quickly enough so that 3 resuming can happen in 2 minutes. There's another rule "DM_binlog_file_gap_between_master_syncer", will the service use it as well?

Also our document should be updated https://docs.pingcap.com/zh/tidb/stable/dm-handle-alerts

Reply:
if we met resumable error for the first time or the first time after long `running` state, `pause/resume` will happen every 5 seconds in default check interval. which type of `pause/resume` is even slower than `connection timeout`?

Reply:
https://github.com/pingcap/tiflow/issues/3734#issuecomment-929070540